tools/sgen/input/FluentUIStyle.yml

```diff
@@ -346,3 +346,17 @@ AP_MSFDrawerTokens:
   cornerRadius: $Border.radius.xxLarge
   verticalShadowOffset: $Spacing.xSmall
   horizontalShadowOffset: $Spacing.small
+
+AP_MSFPersonaGridItemTokens:
+  avatarInterspace: [ small: $Spacing.xSmall, large: $Spacing.small ]
+  backgroundColor: $Colors.Background.neutral1
+  bottomPadding: $Spacing.xSmall
```

Comment:
Should we have padding on the sides here?
I'm mostly thinking of the scenario where the single item is used as a standalone control.

Reply:
Looks nicer too, and will keep the tap zones from having no-tap "gutters" once they're a collection.